plugin.xml

```diff
@@ -86,6 +86,7 @@
 
         <framework src="com.android.support:support-v13:23+" />
         <framework src="com.google.android.gms:play-services-gcm:+" />
+        <framework src="build.gradle" custom="true" type="gradleReference" />
```

Comment:
Instead of including a custom build.gradle file why not use a framework tag:

`<framework src="me.leolin:ShortcutBadger:1.1.4@aar" />`